src/components/InfoPanel.component.js

```diff
@@ -1,8 +1,42 @@
 import React, { Component } from 'react'
+import { Image } from 'react-bootstrap'
+import './../styles/App.css'
 
 class InfoPanel extends Component {
+  constructor(props) {
+    super(props)
+  }
+
   render() {
-    return <div>InfoPanel</div>
+    const selectedEvent = this.props.selectedEvent
+    const img_url = selectedEvent.image
+
+    return (
+      <div className="info-panel">
+        <h1>
+          <u>
+            <b>{selectedEvent.title} </b>
+          </u>
+        </h1>
+        <div>
+          <div>
+            <Image src={selectedEvent.image} alt="H4I image" responsive />
```

Comment:
Right now the `alt` text is okay, but we will need to eventually give it a more descriptive one later. Maybe put a `TODO` comment about it?